Tiltfile

```diff
@@ -299,6 +429,12 @@ def find_object_qualified_name(objs, kind):
             return "{}:{}:{}".format(o["metadata"]["name"], kind, o["metadata"]["namespace"])
     return ""
 
+def find_all_objects_names(objs):
+    qualified_names = []
+    for o in objs:
+        qualified_names = qualified_names + ["{}:{}".format(o["metadata"]["name"], o["kind"])]
```

Comment:
Does this work with namespaced resources? Does it have to work for them?

Reply:
Yeah, this works for namespaces. Where it will fail is if there's multiple resources with the same kind / name in the same objs array.

Reply:
I didn't mean Namespaces (which are cluster-wide), but namespaced resources. I wonder if the name we return has to include the namespace for a namespaced resource (like in find_object_qualified_name).

But based on your response I assume it doesn't have to contain the namespace as long as kind/name is unique for all resources?

I wonder if we should include the namespace for namespaced resources given that it's possible that not every resource is unique across namespaces

Reply:
I'll play around with it during my next review. Maybe I have a suggestion

Reply:
The following works on my machine:
```
        if "namespace" in o["metadata"] and o["metadata"]["namespace"] != "":
          qualified_names = qualified_names + ["{}:{}:{}".format(o["metadata"]["name"],o["kind"],o["metadata"]["namespace"])]
        else:
          qualified_names = qualified_names + ["{}:{}".format(o["metadata"]["name"],o["kind"])]
```

Reply:
(+ the other finding about dropping namespace from the cluster-wide ExtensionConfig CR)